composer.json

```diff
@@ -1,10 +1,75 @@
 {
-  "name": "su-sws/stanford_publication",
-  "description": "A Drupal feature.",
-  "type": "drupal-custom-module",
-  "license": "GPL-2.0+",
-  "minimum-stability": "dev",
-  "require": {
-      "composer/installers": "^1.2"
-  }
+    "name": "su-sws/stanford_publication",
+    "description": "A Drupal publication content type and views.",
+    "type": "drupal-custom-module",
+    "license": "GPL-2.0-or-later",
+    "minimum-stability": "dev",
+    "prefer-stable": true,
+    "repositories": [
+        {
+            "type": "composer",
+            "url": "https://packages.drupal.org/8"
+        },
+        {
+            "type": "package",
+            "package": {
+                "name": "citation-style-language/locales",
+                "version": "1.0.0",
+                "source": {
+                    "type": "git",
+                    "url": "https://github.com/citation-style-language/locales.git",
+                    "reference": "master"
+                }
+            }
+        },
+        {
+            "type": "package",
+            "package": {
+                "name": "citation-style-language/styles-distribution",
+                "version": "1.0.0",
+                "source": {
+                    "type": "git",
+                    "url": "https://github.com/citation-style-language/styles-distribution.git",
```

Comment:
This thing has one branch, no tags, and ~3750 commits. Holy schmokes.

Reply:
Yeah. Apparently it gets auto-synced to another repo after their tests pass.